src/components/layout/Footer.tsx

```diff
@@ -1,18 +1,175 @@
-import { createStyles, makeStyles } from '@material-ui/core'
+import Link from 'next/link'
+import { createStyles, makeStyles, Grid, Container } from '@material-ui/core'
+import { useRouter } from 'next/router'
+import { useTranslation } from 'react-i18next'
+
+import PodkrepiLogo from 'components/brand/PodkrepiLogo'
+import { routes } from 'common/routes'
+
+import { Instagram } from '@material-ui/icons'
+import { Facebook } from '@material-ui/icons'
+import { Twitter } from '@material-ui/icons'
+import { LinkedIn } from '@material-ui/icons'
+import { YouTube } from '@material-ui/icons'
 
 const useStyles = makeStyles((theme) =>
   createStyles({
     footer: {
       position: 'absolute',
-      bottom: 0,
       left: 0,
       right: 0,
-      margin: theme.spacing(3, 0),
+      paddingTop: theme.spacing(11),
+      paddingBottom: theme.spacing(7),
+      backgroundColor: theme.palette.primary.dark,
+    },
+    footer__left: {
+      display: 'flex',
+      flexDirection: 'column',
+    },
+    footer__logo: {
+      width: '70%',
+    },
+    footer__socialContainer: {
+      display: 'flex',
+      flexDirection: 'row',
+      padding: '10px 0',
+    },
+    footer__socialLink: {
+      color: theme.palette.primary.main,
+      marginRight: '5px',
+      cursor: 'pointer',
+    },
+    footer__copyrights: {
+      fontSize: '16px',
+      color: theme.palette.primary.main,
+    },
+    footer__link: {
+      fontSize: '18px',
```

Comment:
Font size is a bit large for my taste, but will leave that as decision for the designers

Reply:
Yes, I agree about the font-size, it's a bit bigger now, so you can change it to 16px - font-size:16px;